BHoM_UI/Components/Engine/GetInfo.cs

```diff
@@ -60,11 +63,58 @@ public class GetInfoCaller : MethodCaller
         /**** Public Method               ****/
         /*************************************/
 
-        [Description("Execute command in the external software")]
-        [Output("information about the current version of the input element (info aboutthe BHoM if no input)")]
-        public static string GetInfo()
+        [Description("Get information about this installed version of the BHoM.")]
+        [MultiOutput(0, "description", "Introduction to what the BHoM is all about.")]
+        [MultiOutput(1, "version", "Version of the BHoM currently running in this interface.")]
+        [MultiOutput(2, "installer", "If you have used the BHoM installer, this gives you information about its exact version.")]
+        [MultiOutput(3, "toolkits", "List of toolkits available in this version of the BHoM.")]
+        [MultiOutput(4, "types", "List of objects types available in this version of the BHoM.")]
+        [MultiOutput(5, "methods", "List of methods available in the version of the BHoM.")]
+        [MultiOutput(6, "adapters", "List of adapters available in the version of the BHoM.")]
+        [MultiOutput(7, "website", "BHoM website.")]
+        [MultiOutput(8, "github", "GitHub organisation containing all our open-source toolkits.")]
+        [MultiOutput(9, "wiki", "Need more details on how to use the BHoM or how to contribute? This is the place to go.")]
+        public static Output<string, string, string, List<string>, List<Type>, List<MethodInfo>, List<Type>, string, string, string> GetInfo()
         {
-            return "This is the BHoM";
+            string description = "This the BHoM (Buildings and Habitats object Model), a collaborative computational development project for the built environment. "
```

Comment:
Should this be `This is the BHoM` rather than `This the BHoM`?

Reply:
done, thanks 😄